app/routes/index.js

```diff
@@ -1,10 +1,14 @@
 import HttpStatus from 'http-status'
 import db from '../../database/models'
+import userRoutes from './user'
 
 const routes = app => {
   // pass the routes to the app for use
   // e.g app.use('/events', eventsRoutes);
 
+  app.use(userRoutes)
+
+
   app.get('/users', async (req, res) => {
```

Comment:
Not part of this pr, but lets move it to a users file in routes i.e routes/users.js

Reply:
sure we will. it was just a dummy route for testing the Models implementation.